src/app/modal/components/lose-changes/lose-changes.component.scss

```diff
@@ -0,0 +1,29 @@
+@import "node_modules/govuk-frontend/govuk/all";
+
+.mat-dialog-content {
+  overflow: hidden;
+}
+
+.title {
+  color: #0B0C0C;
+  font-size: 36px;
+  font-weight: 700;
+  line-height: 43px;
+  width: 705px;
```

Comment:
Does it need to be responsive?
If yes we could use some relative size and use max-width, min-width to lock the ratio